composer.json

```diff
@@ -20,6 +20,7 @@
         "spatie/laravel-permission": "^3.17"
     },
     "require-dev": {
+        "roave/security-advisories": "dev-master",
```

Comment:
What is this?

Reply:
Ditto, could you explain this library @WindOfRussia

Reply:
<img width="873" alt="Screen Shot 2020-10-16 at 2 36 34 PM" src="https://user-images.githubusercontent.com/6550234/96296305-990f3880-0fde-11eb-971a-a6bcbce99ee6.png">

Think of it as a firewall so we don't install bad packages

more info: https://github.com/Roave/SecurityAdvisories